compare.js

```diff
@@ -0,0 +1,8 @@
+const deepEqual = (value1, value2) => {
+    if (typeof (value1) === typeof (value2)) {
+        if (typeof (value1 === "object")) {
+            (JSON.stringify(Object.keys(value1)) === JSON.stringify((Object.keys(value2))) ? true : false)
```

Comment:
You don't need all this junk (ifs) if you are using `JSON.stringifiy`... but don't use it, it's slow and will take into account ordering of the properties. TIP: you'll need some recursion and proper base cases